modules.d/98dracut-systemd/dracut-shutdown-onfailure.service

```diff
@@ -0,0 +1,12 @@
+#  This file is part of dracut.
+#
+# See dracut.bootup(7) for details
+
+[Unit]
+Description=Service executing upon dracut-shutdown failure to perform cleanup
+Documentation=man:dracut-shutdown.service(8)
+DefaultDependencies=no
+
+[Service]
+Type=oneshot
+ExecStart=/bin/sh -c '/bin/rm /run/initramfs/shutdown 2>/dev/null || true'
```

Comment:
```
ExecStart=-/bin/rm /run/initramfs/shutdown
StandardError=null
```
?

Reply:
> ```
> ExecStart=-/bin/rm /run/initramfs/shutdown
> StandardError=null
> ```
> 
> ?

You will need to clarify your ? 

What do you see wrong with the on failiure unit *trying* to execute /bin/rm /run/initramfs/shutdown ( hence the - ) and it dumping it's own output to /dev/null?

How would you solve this differently?

Reply:
It is _my suggestion_ to avoid calling `sh` needlessly...

Reply:
@marcosfrm I somehow manage to miss the entire context of you response, your absolutly correct. @rmetrich can you fix your PR as @marcosfrm suggests or is there any special reason you are escaping to shell to call the command?